docker-compose.yml

```diff
@@ -292,6 +292,18 @@ services:
       - .:/arrow:ro  # ensures that docker won't contaminate the host directory
       - maven-cache:/root/.m2:delegated
 
+  java-all-jdks:
+    # Usage:
+    #   docker-compose build java-all-jdks
+    #   docker-compose run java-all-jdks
+    image: ursalab/arrow-ci-java-all-jdks:latest
```

Comment:
Should we use `ursalabs` (the last `s` is missing) for organization name on Docker Hub?

Reply:
It seems that this name is already taken on Docker Hub. I don't know who owns it

Reply:
Oh. I got it.